web/annotation_layer_builder.css

```
@@ -20,6 +20,7 @@
   --input-unfocused-border-color: transparent;
   --input-disabled-border-color: transparent;
   --input-hover-border-color: black;
+  --link-border: none;
```

Please update this one as well.
```suggestion
  --link-outline: none;
```